### PR TITLE
Create terraform RDS

### LINKED ISSUE #16: Terraform RDS
### Task

- [x] Create terraform folder
- [x] Terraform an RDS 

### User Story 

As a developer, I want to automate the process of creating an RDS on AWS so that I can maintain and repeat this process if need be.

### PR DESCRIPTION
**Pull request by FarihaChoudhury**
Create an AWS RDS using terraform, an infrastructure as code tool, to automate this process.

Closes #16

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+1. Navigate to the project directory:
+
+```bash
+cd terraform
+```
+2. Install the required Python packages:
+
+```bash
+pip install -r requirements.txt
+```
```
Comment (Zhi-704) [inline terraform/README.md]: Do we even have a requirements.txt in the terraform folder?

File: terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#     ingress {
+#         from_port = 5432
+#         to_port = 5432
+#         protocol = "tcp"
+#         cidr_blocks      = ["0.0.0.0/0"]
+#     }
+# }
+
+data "aws_vpc" "c11-vpc" {
+    id = "vpc-04b15cce2398e57f7"
```
Comment (Zhi-704) [inline terraform/main.tf]: Could this not be a terraform variable instead of being hardcoded?

Comment (FarihaChoudhury) [reply terraform/main.tf]: yes I agree, I will make the changes

File: terraform/README.md (hunk 1 above)
Comment (FarihaChoudhury) [reply terraform/README.md]: yes I agree, I will make the changes

File: terraform/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Terraform
+
+This directory focuses on **terraforming** all the AWS cloud services that are used throughout the Railway Tracker application.
+
+
+## Installation
+
+1. Navigate to the project directory:
```
Comment (A-Faris) [inline terraform/README.md]:
```suggestion
Navigate to the project directory:
```

**Review by fm1psy — APPROVED** [review]
This looks good to me, good job

**Pull request merged**